homeassistant/components/zwave_js/const.py

```diff
@@ -30,6 +30,9 @@
 
 LOGGER = logging.getLogger(__package__)
 
+# constants extra state attributes
+ATTR_SPECIAL_VALUES = "special_values"  # ConfigurationValue number entities
```

Comment:
Why is it called special values?

Reply:
that was the first name that came to mind. Open to suggestions on this

Reply:
Ah sorry misunderstood the question - for entities that support manual entry but have states available as well, the idea was to have an attribute that shows the available preconfigured states

Reply:
presets sounds like a good idea. Here's an example just so you get a sense of what I am talking about:
```json
{
  "endpoint": 0,
  "commandClass": 112,
  "commandClassName": "Configuration",
  "property": 2,
  "propertyKey": 255,
  "propertyName": "Tone Play Count (Tamper)",
  "ccVersion": 0,
  "metadata": {
    "type": "number",
    "readable": true,
    "writeable": true,
    "label": "Tone Play Count (Tamper)",
    "default": 1,
    "min": 0,
    "max": 255,
    "states": {
      "0": "Unlimited",
      "255": "Last configuration value"
    },
    "valueSize": 4,
    "format": 1,
    "allowManualEntry": true,
    "isFromConfig": true
  },
  "value": 1
}
```

In this case, the states represent specific configurations that can't be captured by a number. Hence why I called it "special values." But I think presets fits for this use case, what do you think?

Reply:
Either `presets` or `reserved values`?

Reply:
I like reserved values, it fits better than presets 👍